src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -465,7 +465,7 @@ public void TestMaxWorkingSet()
                 Assert.True((long)p.MinWorkingSet >= 0);
             }
 
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX) || RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD")));
```

Comment:
Is it time to add freebsd to the enum now...

Reply:
It exists but only for netcoreapp. We build this also as netstandard.